Refactor modules (#2217)
pyansys/pyaedt: doc/source/API/Post.rst

plots in AEDT. They are accessible through the ``post`` property.

.. note::

>    Some of  `AdvancedPostProcessing` module requires Python 3 and

Reviewer: ```suggestion
   Some capabilities of the ``AdvancedPostProcessing`` module require Python 3 and
```

>    that the `numpy <https://numpy.org/doc/stable/>`_, `matplotlib <https://matplotlib.org/>`_,
>    and `pyvista <https://docs.pyvista.org/>`_ packages are installed.

Reviewer: ```suggestion
   installations of the `numpy <https://numpy.org/doc/stable/>`_, `matplotlib <https://matplotlib.org/>`_,
   and `pyvista <https://docs.pyvista.org/>`_ packages.
```

    standard_report.create()
    sols = standard_report.get_solution_data()
    ...

AEDT report management
~~~~~~~~~~~~~~~~~~~~~~
AEDT provides great flexibility in reports.
PyAEDT has classes for manipulating any report property.

.. note::
   Some functionalities are available only when AEDT is running
   in graphical mode.

.. currentmodule:: pyaedt.modules

.. autosummary::
   :toctree: _autosummary
   :nosignatures:

   report_templates.Trace
   report_templates.LimitLine
   report_templates.Standard
   report_templates.Fields
   report_templates.NearField
   report_templates.FarField
   report_templates.EyeDiagram
   report_templates.Emission
   report_templates.Spectral

Plot fields and data outside AEDT
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> PyAEDT also supports external report capabilities.

Reviewer: ```suggestion
PyAEDT supports external report capabilities.
```

> PyAEDT also supports external report capabilities.
> Those are available with third party packages like  `numpy <https://numpy.org/doc/stable/>`_,
> `pandas <https://pandas.pydata.org/>`_, `matplotlib <https://matplotlib.org/>`_,
> and `pyvista <https://docs.pyvista.org/>`_ packages installed with PyAEDT.

Reviewer: ```suggestion
PyAEDT supports external report capabilities available with installed third-party
packages like `numpy <https://numpy.org/doc/stable/>`_,
`pandas <https://pandas.pydata.org/>`_, `matplotlib <https://matplotlib.org/>`_,
and `pyvista <https://docs.pyvista.org/>`_.
```
